adapters/core2p2p/transaction.go

```diff
@@ -86,17 +86,18 @@ func AdaptTransaction(transaction core.Transaction) *gen.Transaction {
 		case tx.Version.Is(3):
 			specTx.Txn = &gen.Transaction_DeclareV3_{
 				DeclareV3: &gen.Transaction_DeclareV3{
-					Sender:                    AdaptAddress(tx.SenderAddress),
-					Signature:                 AdaptAccountSignature(tx.Signature()),
-					ClassHash:                 AdaptHash(tx.ClassHash),
```

Comment:
`class_hash` was removed in this PR: https://github.com/starknet-io/starknet-p2p-specs/pull/54